src/Common/src/Interop/Windows/mincore/Interop.VersionHelpers.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Kernel32)]
+        private static extern bool VerifyVersionInfo([In] ref OSVERSIONINFOEX lpVersionInfo, uint dwTypeMask, ulong dwlConditionMask);
+
+        [DllImport(Libraries.Kernel32)]
+        private static extern ulong VerSetConditionMask(ulong dwlConditionMask, uint dwTypeBitMask, byte dwConditionMask);
```

Comment:
When I tested this on my Windows 10 PC, this returned true for Windows 8.1 or Greater and false for Windows 10 or Greater. It seems like this is similar to GetVersion, only difference is this API doesn't lie, Windows 10 is technically greater than Windows 8.1, whereas ```GetVersion``` would have returned Windows 8.1 on Windows 10.